scripts/open_url_in_instance.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2019 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+# Copyright 2019 Gart Maddy (NoSuck) <admin@nosuck.org>
+
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Script for faster execution.
+
+This script greatly speeds up execution of qutebrowser commands when a
+qutebrowser instance already exists.  Call it in place of your system's
+qutebrowser executable.
+"""
+
+import hashlib
+import json
+import os
+import os.path
+import socket
+import subprocess
+import sys
+
+version = '1.6.0'
+protocol_version = 1
+
+m = hashlib.md5()
+m.update(os.environ['USER'].encode('utf8'))
```

Comment:
My earlier suggestion about using `os.path.expanduser('~')` indeed was wrong, sorry about that! What I meant was this:

```suggestion
m.update(getpass.getuser())
```

It's the same what qutebrowser itself does in `ipc.py`, and is more platform-independent.

(You'll also need to add the import)

Reply:
Ahh, thank you.  I got a new Python battery today.